demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -335,7 +329,8 @@ def run_validations_on_file(self, file_path, pack_error_ignore_list, is_modified
         elif file_type == FileType.BETA_INTEGRATION:
             return self.validate_beta_integration(structure_validator, pack_error_ignore_list)
 
-        elif file_type == FileType.IMAGE:
+        # For old integration in which the image is inside the yml file.
+        elif file_type in (FileType.IMAGE, FileType.INTEGRATION, FileType.BETA_INTEGRATION):
             return self.validate_image(file_path, pack_error_ignore_list)
```

Comment:
this is incorrect for several reasons:
1) `run_validations_on_file` will only send the file to a single "validate" method - so an Integration should not go to an image validator - this goes against the design.

2) Integration validator checks images, see `is_valid_image` method in demisto_sdk/commands/common/hook_validations/integration.py

you can do one of the following:
1) revert this change - let images in integrations be checked through integrations as it does right now.
2) revert this change but add a call to image validator in `validate_integration` and `validate_beta_integration` methods and then remove the image validation from the integration validator. (if you are doing this, please do the same to descriptions for consistency)

Reply:
I checked and you are right, wasnt aware of the image validation in integrations.
Will revert the change then.